Resources/Locale/en-US/holopad/holopad.ftl

```diff
@@ -66,6 +67,7 @@ holopad-command-qm = Command - QM
 holopad-command-ce = Command - CE
 holopad-command-rd = Command - RD
 holopad-command-hos = Command - HoS
+holopad-command-evac = Command - Evac
```

Comment:
What station has Evac as a command room?

Reply:
Downstream. We have several of them.